av/filter/graph.pyx

```diff
@@ -122,7 +123,7 @@ cdef class Graph(object):
             self._register_context(py_ctx)
         self._nb_filters_seen = self.ptr.nb_filters
 
-    def add_buffer(self, template=None, width=None, height=None, format=None, name=None):
+    def add_buffer(self, template=None, width=None, height=None, format=None, time_base=None, name=None):
```

Comment:
Why change the order of the arguments? In a future release we might switch to keyword-only arguments, but this seems like a gratuitous breaking API change.